### PR TITLE
style(content-section): fixing theme error on storybook

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Related Ticket(s)

[Content Section: Theme issue in Storybook: On applying g100, g90 , g10 text in children section is not visible #1656](https://github.com/carbon-design-system/ibm-dotcom-library/issues/1656)

### Description

The themes were not being applied correctly. This problem seems to related only with the storybook appearance.

### Changelog

**Changed**

- `packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss`

### CODE REVIEW

File: packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss [hunk 1]
```diff
@@ -1,7 +1,30 @@
 @import '../../../../../../styles/scss/patterns/sub-patterns/content-section/content-section';
 
+@mixin themed-items {
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss:3]: Let's rename this to something specific to the content section, perhaps `content-section-theme-items`. Remember that mixins are available outside of this file if compiled.

Comment (guilhermelMoraes) [reply packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss]: Ok, all right. I did it

File: packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+@mixin content-section-themed-items {
+  color: $text-01;
+  background: $ui-01;
+}
+
 .#{$prefix}--content-section-story .#{$prefix}--content-section__children {
   height: carbon--mini-units(60);
-  background-color: #f4f4f4;
   padding: $carbon--spacing-05;
+  @include themed-items;
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss:11]: @guilhermelMoraes you'll have to update these includes too...

Comment (guilhermelMoraes) [reply packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss:11]: My bad!

**Pull request merged**